datajoint/autopopulate.py

```diff
@@ -16,28 +17,27 @@ class AutoPopulate(metaclass=abc.ABCMeta):
     """
     AutoPopulate is a mixin class that adds the method populate() to a Relation class.
     Auto-populated relations must inherit from both Relation and AutoPopulate,
-    must define the property populated_from, and must define the callback method _make_tuples.
```

Comment:
When I write tutorials and record presentations, I have to use `poprel` as a noun and it's much easier to discuss than `populated_from`.  It's a new term that we introduce in datajoint, which has specific uses. It has been used in MATLAB from the early versions, so changing it there is not practical.  Naming it the same way in both languages helps.